include/boost/geometry/io/wkt/detail/prefix.hpp

```diff
@@ -42,6 +47,11 @@ struct prefix_linestring
     static inline const char* apply() { return "LINESTRING"; }
 };
 
+struct prefix_polyhedral_surface
+{
+	static inline const char* apply() { return "POLYHEDRALSURFACE Z"; }
```

Comment:
Not sure about the `Z` as per @barendgehrels  https://github.com/boostorg/geometry/pull/782#discussion_r550097589